packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -112,25 +147,37 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
           const delegatorAddress = this.state.chainAddress;
 
           const result = await E(account).executeEncodedTx([
-            /** @type {AnyJson} */ (
-              Any.toJSON(
-                MsgDelegate.toProtoMsg({
-                  delegatorAddress,
-                  validatorAddress,
-                  amount,
-                }),
-              )
+            toJSON(
+              MsgDelegate.toProtoMsg({
+                delegatorAddress,
+                validatorAddress,
+                amount,
+              }),
             ),
           ]);
 
           if (!result) throw Fail`Failed to delegate.`;
-          try {
-            const decoded = MsgDelegateResponse.decode(decodeBase64(result));
-            if (JSON.stringify(decoded) === '{}') return 'Success';
-            throw Fail`Unexpected response: ${result}`;
-          } catch (e) {
-            throw Fail`Unable to decode result: ${result}`;
-          }
+          return tryDecodeResponse(result, MsgDelegateResponse.fromProtoMsg);
+        },
+
+        /**
+         * @param {CosmosValidatorAddress} validator
+         * @returns {Promise<ChainAmount[]>}
+         */
+        async withdrawReward(validator) {
+          const { chainAddress } = this.state;
+          const msg0 = MsgWithdrawDelegatorReward.toProtoMsg({
```

Comment:
the `0` is distracting. consider removing or making a more explicit message name